stylelint-polaris/index.js

```diff
@@ -47,243 +44,25 @@ const disallowedVarsBorder = [
   '--p-banner-border-warning',
   '--p-banner-border-critical',
   '--p-text-field-focus-ring-offset',
-  '--p-border-base',
-  '--p-border-dark',
-  '--p-border-transparent',
-  '--p-border-divider',
-  '--p-border-divider-on-dark',
-];
-
-const disallowedVarsColor = [
-  // Legacy custom properties
-  '--p-override-transparent',
-  '--p-badge-mix-blend-mode',
```

Comment:
```js
'--p-override-transparent',
'--p-badge-mix-blend-mode',
```
While these two tokens are not deprecated v10 custom properties they can be removed since they are no longer present in polaris or web ([see audit here](https://github.com/Shopify/polaris/issues/7622)).